spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -373,6 +383,7 @@ def main(argv: Sequence[str]):
 
     file_out = os.path.abspath(arguments.o)
     append = bool(arguments.append)
+
```

Comment:
Nit: There are a few of these whitespace changes remaining in the file. Could you please revert them to minimize the amount of noise in the PR? 

(See also: https://medium.com/google-developer-experts/how-to-pull-request-d75ac81449a5#e07e)

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/f35f8a13d0aa8bb9096ca8af49b60337389f4531

Reply:
Sorry to nitpick more, but I think only one of the whitespace lines was fixed -- there are 3 more in the file, hehe. :sweat_smile:

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/1e56d4d334f5d4476a8d56c4d20e932faa403348